Changelog.md

```diff
@@ -1,5 +1,47 @@
 # Changelog
 
+### 1.1.1
+Changes:
```

Comment:
Is it possible to modify the previous format?

```
**Image:**
  - k8s.gcr.io/ingress-nginx/controller balabala
```

Reply:
My bad. I thought I did put the "Image" section there. Now added. Also added 2 sentences summarising the release. Please provide any other text, that could go into the release-description/release-notes.